### ISSUE TITLE
Only Nether Fortress Mobs Spawning

### ISSUE THREAD

**Issue by FlexRig** [Cazadorsniper/PO3#1664]
# Use the search function before reporting to see if the issue has been reported before.
**Pack Version:** 3.4.8
**Pack Mode (Normal, Titan, Kappa):** Kappa
**Singleplayer, Multiplayer, or LAN?:** Multiplayer

**Description:**
When we first started playing and entered the Nether, we had many different mobs such as the black widows, ghasts, salamanders, etc.  spawning. For some reason, all mobs, other than Nether fortress mobs, have stopped spawning. Pigmen (at Nether Fortresses), Wither Skeletons, Blazes, and Magma Cubes are the only mobs spawning. Magma Cubes do spawn at the slime islands though. Nethermites also spawn within some blocks. We have tried the /cofh killall command and it does show that there were some other mobs somewhere within the dimension. Picture of that attached.

**Steps to reproduce:**

**Crash log (use pastebin or issue will be closed):**
![unknown](https://user-images.githubusercontent.com/108161477/175752991-89dcd00b-960c-4fc4-8926-66300ac01845.png)

**Comment by FlexRig** [comment]
To add some more info, all mob farms have been disabled. It seemed to occur after one of our friends first made a mob farm. They have been shut down since then and all chunks have been unclaimed. Furthermore, the amount of Nether Fortress mobs spawning is insane. Whenever you come across a Nether Fortress, there's immediately a ton of mobs.

**Comment by Cazadorsniper** [comment]
Seems quite weird, not sire what it could be though. Could try killing all mobs in the nether maybe.

**Comment by FlexRig** [comment]
Yeah we tried that. Also did a fresh install of the server files and copied the world over.

**Comment by Crush3rX** [comment]
Hi, we are experiencing the same on a multiplayer server, from one day to another all mobs in nether disappeared. They only spawn on nether fortresses. Did you find a solution @FlexRig or @Cazadorsniper ?

**Comment by Cazadorsniper** [comment]
No solution found. Not sure what the cause is.

**Comment by Crush3rX** [comment]
> No solution found. Not sure what the cause is.

Would it help you troubleshoot if I would upload my server files for you to see? we havent changed any configs besides setting allow-flight to true and spawn-protection to 0

**Comment by Cazadorsniper** [comment]
just send the actual world itself.

**Comment by FlexRig** [comment]
@Crush3rX No we still haven't found a solution. We reinstalled the entire server with no change. My buddy's going to load up in singleplayer and see if it's broken there.

**Comment by FlexRig** [comment]
> just send the actual world itself.

Can we send you our world to see if you can find a problem?

**Comment by Cazadorsniper** [comment]
You can try resetting the nether.

**Comment by FlexRig** [comment]
> You can try resetting the nether.

He's reset the Nether and even made a fresh server and imported the world without the Nether. Still didn't fix it. Furthermore, I haven't spent an extreme amount of time looking, but I think because of this, there's no little temple or whatever to summon the Ghast Queen.

**Comment by FlexRig** [comment]
@Crush3rX Did you guys ever come up with anything?

**Comment by Doodyk** [comment]
Hey, I have also started experiencing this problem, I had a moment where everything stopped spawning after using a mekanism command that seemingly has no documentation "/mek testrules" and took me a while to figure out how to fix it, not sure if that's the cause of the lack of mobs spawning as I haven't seen nether mobs since I ran that command. Im on pack version 3.4.10 as a heads up

**Comment by LDrender** [comment]
Hey, I have also started experiencing this problem.

**Comment by ios777** [comment]
i found a temporary solution. i just create a new creative world. i unlocked the nether and used a nether cake to go there. All the hostile mob were spawning. then i went back to my main world (i was already in the nether) and it started working properly.
good luck

**Comment by ios777** [comment]
Didn't last for long.
Now switching betwheen the two world seems to solve it. i'm always in the nether when i change between savefile

**Comment by Cazadorsniper** [comment]
I can only think of Netherex doing that to the nether.
